generator/generatorsetqtscript.h

```diff
@@ -62,6 +62,7 @@ class GeneratorSetQtScript : public GeneratorSet
 
 private:
     MetaQtScriptBuilder builder;
+    int maxClassesPerFile{30};
```

Comment:
I wonder if we should use a large number as default instead to make changes by default better comparable. I assume that the restriction had to do with limitations / performance issues with earlier compilers and IDEs, and I doubt that this is still needed nowadays. Would like to hear other opinions...